packages/switch/src/react/__stories__/index.story.tsx

```diff
@@ -43,64 +42,71 @@ storiesOf('click', module)
   ))
 
 const labelStory = storiesOf('label', module)
-Object.keys(Switch.sizes).forEach(size =>
-  Object.keys(Switch.labelAligns).forEach(labelAlign =>
-    labelStory.add(`${size} ${labelAlign}`, _ => (
+Object.values(Switch.sizes).forEach(size =>
+  Object.values(Switch.labelAligns).forEach(labelAlign =>
+    labelStory.add(`${size} ${labelAlign}`, () => (
       <Switch size={size} labelAlign={labelAlign}>
         Click me
       </Switch>
     ))
   )
 )
 
+type RenderProp = {
```

Comment:
nit: It's test, perhaps less important...

"RenderProp" seems to misname what this is.  The render prop is the function given to the parent component.  These are more like "RenderPropProps".